src/Surfnet/StepupGateway/GatewayBundle/Entity/SamlEntityRepositoryInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupGateway\GatewayBundle\Entity;
+
+interface SamlEntityRepositoryInterface
```

Comment:
As far as I know, we haven't used the `Interface` suffix thus far. Could you change this, for consistency's sake?